### ISSUE TITLE
Problem with key extraction, edit.title is extracted as edititle

### ISSUE THREAD

**Issue by mme-private** [ngneat/transloco-keys-manager#7]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

After running transloco-keys-manager extract there is wrong key generated.

I have added lazy loading module:
```ts
@NgModule({
	imports: [
		...
		TranslocoModule,
	],
	declarations: [
		...
	],
	providers: [{ provide: TRANSLOCO_SCOPE, useValue: 'property' }],
})
export class PropertyModule { }
```

And have component template:
```html
<ng-container *transloco="let t; read: 'property'">
...
			<h3 class="modal-title">{{ t('edit.title') }}</h3>
			<div class="modal-body">
...
```

After running transloco-keys-manager extract, there is key generated (assets/i18n/property/en.json):
```json
{
  "edititle": "Missing value for 'property.edititle'",
}
```

Tried workaround:
```html
<ng-container *transloco="let t; read: 'property'">
	...
			<ng-container *transloco="let t; read: 'property.edit'">
				<h3 class="modal-title">{{ t('title') }}</h3>
			</ng-container>
        ...
```

Which generated correct key, but also generated two keys:
```json
{
  "edit.title": "Missing value for 'property.edit.title'",
  "title": "Missing value for 'property.title'"
}
```

## Expected behavior

Key should be named: "edit.title"

## Environment

<pre><code>
Angular version: 8.2.12

For Tooling issues:
- Node version: v12.13.1
- Platform:  Windows 7 Ultimate

Others:

</code></pre>

**Comment by NetanelBasal** [comment]
We'll release a fix in the next release, thanks.

**Comment by shaharkazaz** [comment]
@mme-private This will be available in v1.2.0 that will be released tomorrow morning 🙂
